### PR TITLE
🔒️ Make Redis Client support TLS 

### PR DESCRIPTION
**Pull request by YuryHrytsuk**
## What do these changes do?

Make Redis Client support TLS via `REDIS_SECURE` field

## Related issue/s

- unblocks https://github.com/ITISFoundation/osparc-ops-environments/issues/241

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))
  * https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/867
  * Activate aws hosted Redis --> https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/868

### CODE REVIEW

**Review by sanderegg — APPROVED** [review]
~~great~~ mesmerizing :+1:  thanks

**Review by matusdrobuliak66 — APPROVED** [review]
thanks!

File: packages/settings-library/src/settings_library/redis.py [hunk 1]
```diff
@@ -30,7 +31,7 @@ class RedisSettings(BaseCustomSettings):
 
     def build_redis_dsn(self, db_index: RedisDatabase):
         return RedisDsn.build(
-            scheme="redis",
+            scheme="rediss" if self.REDIS_SECURE else "redis",
```
[the remark this answers is not in the document]
Comment (pcrespov) [reply packages/settings-library/src/settings_library/redis.py:34]: MINOR: what about stressing that we append an `s`?
```python
"redis" + "s" if self.REDIS_SECURE else ""
```
I would even have thought it was a typo! :-)

**Review by pcrespov — APPROVED** [review]
thxssss ;-)

**Review by GitHK — APPROVED** [review]
👍

**Pull request merged**